### PR TITLE
API phpunit9 support

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10019

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         "silverstripe/asset-admin": "^1.0"
     },
     "require-dev": {
-        "sminnee/phpunit": "^5.7",
+        "phpunit/phpunit": "^9",
```
Comment (maxime-rainville) [inline composer.json:18]:
```suggestion
        "phpunit/phpunit": "^9.5",
```

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Looks good, just need to make the PHPUnit constraint `^9.5`. Maybe add the PHP constraint while you're in there.

File: composer.json (hunk 1 above)
Comment (emteknetnz) [reply composer.json]: Updated

**Review by maxime-rainville — APPROVED** [review]
LGTM

**Pull request merged**